cli/scripts/bin/boomi_runtime_installer.sh

```diff
@@ -69,9 +71,10 @@ sudo apt-get install -y libxml2-utils -y
 
 mkdir -p  /home/$USR/boomi/boomicicd
 cd /home/$USR/boomi/boomicicd
-echo "git clone https://github.com/UnitedTechnoCloud/boomiinstall-cli..."
-git clone https://github.com/UnitedTechnoCloud/boomiinstall-cli
+echo "git clone https://github.com/$GitOrgName/boomiinstall-cli..."
+git clone --single-branch --branch changes_for_aws https://github.com/${GitOrgName}/boomiinstall-cli
```

Comment:
Why do we need to git clone from the private repo? the same files already exist in the public repo. Let's discuss next time?